sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3630,6 +3630,35 @@ object functions {
     window(timeColumn, windowDuration, windowDuration, "0 second")
   }
 
+  /**
+   * Generates session window given a timestamp specifying column.
+   *
+   * Session window is the one of dynamic windows, which means the length of window is vary
```

Comment:
Session window is the one of dynamic windows -> Session window is one of dynamic windows

Reply:
the length of window is vary -> the length of window is varying